src/assets/styles/main.scss

```diff
@@ -19,17 +19,26 @@ $icon-font-path: '../fonts' !default;
   line-height: 1.5em;
 }
 
+
+// Need to set base font size for em to all cloudinary components
+.cld-video-player,
```

Comment:
@gilnoycloudinary as a reminder - need to make sure that light skin works correctly.